src/pkcs11/framework-pkcs15.c

```diff
@@ -5280,6 +5280,11 @@ pkcs15_profile_get_attribute(struct sc_pkcs11_session *session, void *object, CK
 		check_attribute_buffer(attr, sizeof(CK_ULONG));
 		*(CK_ULONG*)attr->pValue = pobj->profile_id;
 		break;
+	case CKA_TOKEN:
+		/* NSS test for this */
```

Comment:
I think we can drop the NSS specific comment here, as we agree that this property is described (and enforced) well enough in the standard.

Reply:
@dengert can you update the PR removing this comment so we can merge this?

Reply:
Forced push without comment.